avocado/core/jobdata.py

```diff
@@ -170,7 +170,11 @@ def get_resultsdir(logdir, jobid):
     """
     Gets the job results directory using a Job ID.
     """
-    if jobid == 'latest':
+    if os.path.isdir(jobid):
+        return os.path.expanduser(jobid)
+    elif os.path.isfile(jobid):
```

Comment:
Actually if I were to specify path to a file, I'd expect it to be the `id` file. But in the end I'd rather remove the support for this as it adds the possibility of specifying local file instead of hash.